cmd/insprd/operators/kafka/utils.go

```diff
@@ -1,4 +1,4 @@
-package channels
+package kafka
```

Comment:
The "main" kafka lib ("github.com/confluentinc/confluent-kafka-go/kafka") is used by using `kafka.Something`, so I think it's better to rename this package to `kafkaop`or something alike